src/test/java/uk/ac/cam/cl/dtg/util/mappers/EventMapperTest.java

```diff
@@ -0,0 +1,205 @@
+package uk.ac.cam.cl.dtg.util.mappers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static uk.ac.cam.cl.dtg.util.mappers.MapperTestUtils.assertDeepEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.joda.time.Instant;
```

Comment:
We shouldn't be using the Joda library anymore. We even have a ticket to remove it. Instead, use `java.time`.